docs/6.x/cluster.md

```diff
@@ -1464,6 +1464,54 @@ The command can also output the images in the json or yaml format which can come
 $ gravity registry list --format=json
 ```
 
+## Changing Node Advertise Address
+
+!!! note "Supported version":
+    Changing a node's advertise address is supported starting from Gravity 7.
```

Comment:
I think we should fork our docs and put this into 7.0 vs putting it into 6.x. Unless this feature is party back-ported to an earlier version.

Reply:
Yeah, we should. Not as a part of this PR though.